vendor/k8s.io/kubernetes/Godeps/Godeps.json

```diff
@@ -1462,6 +1558,34 @@
 			"ImportPath": "honnef.co/go/tools",
 			"Rev": "v0.0.1-2019.2.2"
 		},
+		{
+			"ImportPath": "k8s.io/api",
+			"Rev": "v0.18.0-beta.2"
```

Comment:
does it look good?

Reply:
@marun we should do another rebase to final version as soon as possible. Basically same procedure, only that picking into kube fork will be trivial, and tests most probably stay green.

Reply:
rc.1. Why beta.2?

Reply:
ok, for whatever reason (you discussed with @marun if I have understood correctly) all are beta.2 here. So it fine for api as well.